qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -2,7 +2,7 @@
     virt_test_type = qemu
     login_timeout = 360
     type = pci_bridge
-    only i440fx pseries
+    only i440fx pseries aarch64
```

Comment:
i440FX and pseries use `PCI` bridge, but `aarch64` uses `PCIE` bridge, so this cfg is not applicable to aarch64.

Reply:
You can see that aarch64 supports this device, and it works in cmd.
```
[root@ampere-hr330a-13 tests]# /usr/libexec/qemu-kvm -device pci-bridge,?
pci-bridge options:
  addr=<int32>           - Slot and optional function number, example: 06.0 or 06 (default: -1)
  bus-reserve=<uint32>   -  (default: 4294967295)
  chassis_nr=<uint8>     -  (default: 0)

```

```
    -device pcie-root-port,id=pcie-root-port-1,port=0x1,addr=0x1.0x1,bus=pcie.0,chassis=2 \
    -device virtio-gpu-pci,bus=pcie-root-port-1,addr=0x0 \
    -device pci-bridge,id=pci_bridge0,bus=pcie.0,addr=0x2,chassis_nr=1 \
```
Of course, we only test from pcie-root-port when testing hotplug, so I banned related cases. But I think it is possible to bring this -device to boot and migrate.

What do you think?

Reply:
Not exactly. `q35` also supports pci-bridge, but we wouldn't test it, so you can see another test scenario named `pcie_controllers.cfg` for `q35`.

Reply:
Thanks a lot, I agree with you, I will investigate `pcie_controllers.cfg` next.